force_wfmanager/central_pane/plot.py

```diff
@@ -1,11 +1,15 @@
 from chaco.api import ArrayPlotData, ArrayDataSource, ScatterInspectorOverlay
 from chaco.api import Plot as ChacoPlot
+from chaco.api import BaseXYPlot, ColormappedScatterPlot
+from chaco.default_colormaps import (
+    color_map_functions, discrete_color_map_functions
+)
 from chaco.tools.api import PanTool, ScatterInspector, ZoomTool
 from enable.api import Component, ComponentEditor
 from enable.api import KeySpec
 from traits.api import (
     Button, Bool, Enum, HasStrictTraits, Instance, List, Property, Tuple,
-    on_trait_change
+    on_trait_change, Dict,
```

Comment:
I think alphabetising (sp?) imports are either an enthought style thing, or possibly just something Mark told me to do once! It does help prevent git making merge conficts so it's not just pedantry haha

Reply:
That's a good tip, thanks!